### DISCUSSION TITLE
Video Card Question

### DISCUSSION THREAD

**Discussion by EricWvi** [osy86/HaC-Mini#589] [Q&A] [answered]
How can I know if the graphics card AMD Radeon RX Vega works well, say when I play games?
![image](https://user-images.githubusercontent.com/49674577/119641199-fceac100-be4b-11eb-84ad-126d01e173f2.png)

**Comment by Medizinmann-MD** [comment, marked as answer]
You could run activity monitor in MacOS and look at the GPUs activity.

Or run a Metal benchmark - or both...;-)
